index.js

```diff
@@ -1,9 +1,10 @@
 /**
  * @format
  */
+import 'react-native-gesture-handler';
```

Comment:
I'm not sure this import is useful ?

Reply:
It is mandatory for the slider (cf https://reactnavigation.org/docs/drawer-navigator#installation)